content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -102,13 +102,22 @@ to ensure that the API presents a clear, consistent view of system resources
 and behavior, and to enable controlling access to end-of-life and/or
 experimental APIs.
 
-Refer to [API versions reference](/docs/reference/using-api/api-overview/#api-versioning)
-for more details on the API version level definitions.
-
 To make it easier to evolve and to extend its API, Kubernetes implements
 [API groups](/docs/reference/using-api/api-overview/#api-groups) that can be
 [enabled or disabled](/docs/reference/using-api/api-overview/#enabling-or-disabling).
 
+API resources are distinguished by their API group and their resource types
```

Comment:
kind is not exactly the same as resource. I would say:

> API resources are distinguished by their API group, resource type, namespace (for namespaced resources), and name. The API server may serve the same underlying data via multiple versions, and handles conversion between API versions transparently.

Reply:
👋  @liggitt , To clear up my understanding,
Does the value of a resource's `kind` field represent the resource type?

Reply:
It is related, but not 1:1. The same kind can map to multiple resource URLs. For example, for resources in the core API group in the v1 version:

```
kubectl get --raw /api/v1 | jq -r '.resources[] | "kind=\(.kind), resource=\(.name)"' | sort
kind=Binding, resource=bindings
kind=Binding, resource=pods/binding
kind=ComponentStatus, resource=componentstatuses
kind=ConfigMap, resource=configmaps
kind=Endpoints, resource=endpoints
kind=Event, resource=events
kind=Eviction, resource=pods/eviction
kind=LimitRange, resource=limitranges
kind=Namespace, resource=namespaces
kind=Namespace, resource=namespaces/finalize
kind=Namespace, resource=namespaces/status
kind=Node, resource=nodes
kind=Node, resource=nodes/status
kind=NodeProxyOptions, resource=nodes/proxy
kind=PersistentVolume, resource=persistentvolumes
kind=PersistentVolume, resource=persistentvolumes/status
kind=PersistentVolumeClaim, resource=persistentvolumeclaims
kind=PersistentVolumeClaim, resource=persistentvolumeclaims/status
kind=Pod, resource=pods
kind=Pod, resource=pods/log
kind=Pod, resource=pods/status
kind=PodAttachOptions, resource=pods/attach
kind=PodExecOptions, resource=pods/exec
kind=PodPortForwardOptions, resource=pods/portforward
kind=PodProxyOptions, resource=pods/proxy
kind=PodTemplate, resource=podtemplates
kind=ReplicationController, resource=replicationcontrollers
kind=ReplicationController, resource=replicationcontrollers/status
kind=ResourceQuota, resource=resourcequotas
kind=ResourceQuota, resource=resourcequotas/status
kind=Scale, resource=replicationcontrollers/scale
kind=Secret, resource=secrets
kind=Service, resource=services
kind=Service, resource=services/status
kind=ServiceAccount, resource=serviceaccounts
kind=ServiceProxyOptions, resource=services/proxy
kind=TokenRequest, resource=serviceaccounts/token
```

Note how the same kind can be sent to multiple resource endpoints. Tools like kubectl submit a given kind to the *first* resource that matches the kind in the manifest file.